### DISCUSSION TITLE
AdGuard Crashing !!!!

### DISCUSSION THREAD

**Discussion by muhanadali100** [AdguardTeam/AdGuardHome#2747] [Q&A]
I used ADGuard for 1 Week after last update the ADGuard Crash and i don't know why
![Screenshot_2021-02-19 AdGuard Home(1)](https://user-images.githubusercontent.com/31727156/108545955-804ec080-72f9-11eb-9e21-7cffeaf7921b.png)
![Screenshot_2021-02-19 AdGuard Home](https://user-images.githubusercontent.com/31727156/108545962-82b11a80-72f9-11eb-8c68-59584a6a147a.png)

**Comment by Aikatsui** [comment]
Follow the bug report template or you can open new issue. 

Provide us AdGuard Home logs.

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
2. Reproduce the issue.
3. Post the log file here.
